app/assets/styles/styles.css

```diff
@@ -38,29 +39,25 @@ p {
 }
 
 .language-menu--item:hover {
-  color: #ffc600;
+  color: #fdfd56;
 }
 
 .primary-nav {
 	margin: 0.4em auto 0;
 	text-align: center;
 }
 
-.logo-img {
-	height: 90px;
-	width: auto;
-}
-
 .logo-brand {
-	color: #091342;
-	font-family: 'Baskerville Old Face'; 
+	color: #1a1451;
+	font-family: 'Trajan Pro Bold'; 
```

Comment:
Where are the font files? Is this a web font that's included with the browsers?  Also this is missing fallback fons.

Reply:
Ups, al parecer está incluido en el browser.

Reply:
It appears you'll need to download and install the font. Just install the font-weight that we need. Don't forget to add fallbacks and the font-family `serif`.